pkg/cmd/cli/cmd/login.go

```diff
@@ -79,6 +85,13 @@ func NewCmdLogin(fullName string, f *osclientcmd.Factory, reader io.Reader, out
 				os.Exit(1)
 
 			} else {
+				if os.IsPermission(err) {
+					fmt.Fprintf(out, "The kubeconfig file %q is not writable. Specify a writeable path with --config or by setting $KUBECONFIG.\n", options.PathOptions.GetDefaultFilename())
+					if Username == "system:admin" {
+						fmt.Fprintln(out, "Warning: Please do not use system configuration file. This can lead to serious errors.")
+					}
+					os.Exit(1)
```

Comment:
We still shouldn't be doing this here -- allow the `kcmdutil.CheckErr` to handle exits.